srt-transmit/src/helptext.txt

```diff
@@ -1,78 +1,88 @@
 Supported protocols:
 
-UDP - send over a UDP port. 
+ TCP - send over a TCP connection
+    example:
+        srt-transmit
+            tcp://host:1234 \
+                # ^- connect to host:1234 \
+            tcp://host:1234?listen
```

Comment:
This is not how I've exposed it for SRT (I wanted to match the reference impl, although i'm not sure what they do for TCP), what we do is the host part of the URL is always a remote host, so if left out it's assumed that you want to bind to a local port. I added the `interface` query parameter to be able to specify what local interface to bind to (if not 0.0.0.0)

Reply:
that I know the `srt-*-transmit` support srt, udp and file.

```
Supported schemes:
	srt: use HOST, PORT, and PARAM for setting socket options
	udp: use HOST, PORT and PARAM for some UDP specific settings
	file: only as file://con for using stdin or stdout
```

I'll change the scheme handling to match what you use for the other protocols

Reply:
Done.